pkg/webhook/github.go

```diff
@@ -138,6 +138,17 @@ func (s *githubHook) handleCheck(c *gin.Context, eventType string) {
 		return
 	}
 
+	if proj.SharedSecret == "" {
+		c.JSON(http.StatusInternalServerError, gin.H{"status": "No secret is configured for this repo."})
```

Comment:
If I recall the steps to create an app + docs from GitHub, the secret is optional on their side. 

Maybe we should also update our docs and clearly state that this is not optional?

Reply:
It is optional for github but we require it for brigade.

Here is the doc: https://github.com/adamreese/brigade-github-app/blob/master/README.md#6-add-brigade-projects-for-each-github-project

Reply:
We could make it more clear